### PR TITLE
ISSUE 4613 - Now gotoviewpoint only shows it if there's relevant data

### LINKED ISSUE #4613: No viewpoint ticket overrides meshes visibility
### Description
When clicking a ticket which viewpoint was deleted it shows all the meshes

- This is happening in: both 
- This requires a hotfix to production: yes

### Steps to replicate
- Open a container or federation
- Select any or all meshes hide them and create a view
- Click on show all
- Create a ticket with a default viewpoint and click on "save current view" in the default view field
- Go back to ticket list and unselect the ticket
- After saved, delete the default viewpoint
- Click on the the created view with a hidden mesh
- Select the ticket with no apparent default view

### Current Behaviour
- The viewer shows the hidden meshes

### Expected Behaviour
- Selecting a ticket with a deleted group shouldnt afect the current view

### Pictures/ Gifs of the error

hidden-bug

### PR DESCRIPTION
**Pull request by sanmont3drepo**
This fixes #4613 

#### Description
- When you try to show a viewpoint and it has no groups and no camera and no clipping planes, just ignore it.

### CODE REVIEW

**Review by carmenfan — CHANGES_REQUESTED** [review]
Found some bugs:

### Bug 1
- Go to this model : https://issue-4613.dev.3drepo.io/v5/viewer/DevTeamWorkspace/7b6c7bd0-054a-11ee-ac5b-59e952832776/0aed56a0-2d3f-11ee-8c0b-17a4bf167a31
- Click on `View 7` which hides some objects and colour overrides a tree
- Then click on the ticket `MNV:2 - test`

The ticket only has a camera, no groups - view moves to the right place and resets the hidden objects, but the tree is still pink

### Bug 2
Try to view `MNV:3`, it's white screening: https://issue-4613.dev.3drepo.io/v5/viewer/DevTeamWorkspace/7b6c7bd0-054a-11ee-ac5b-59e952832776/0aed56a0-2d3f-11ee-8c0b-17a4bf167a31

img-28-47-23-11-47

**Review by carmenfan — APPROVED** [review]

File: frontend/src/v5/helpers/viewpoint.helpers.ts [hunk 1]
```diff
@@ -191,12 +192,18 @@ export const toColorAndTransparencyDicts = (overrides: GroupOverride[]): Overrid
 };
 
 export const goToView = async (view: Viewpoint) => {
+	if (isEmpty(view?.state?.colored)  && isEmpty(view?.state?.hidden) && isEmpty(view?.camera) && isEmpty(view?.clippingPlanes)) {
```
Comment (Amantini1997) [inline frontend/src/v5/helpers/viewpoint.helpers.ts:195]: Given this, I think you can delete line `108` in TicketDetailsCard.component.tsx (`if (isEmpty(defaultView)) return;`)

**Review by Amantini1997 — CHANGES_REQUESTED** [review]

**Review by Amantini1997 — APPROVED** [review]

**Pull request merged**